Add Mappers' Report: February newspost (#3233)
ppy/osu-wiki: news/2020-02-18-mappers-report-february.md

[Yales](https://osu.ppy.sh/users/2377881)

**In osu!catch:** 
[Lacrima](https://osu.ppy.sh/users/4915649) & 
[Dapuluous](https://osu.ppy.sh/users/8140944)

**In osu!mania:** 
[Voxa](https://osu.ppy.sh/users/9229539), 
[Dubstek](https://osu.ppy.sh/users/9555243), 
[UnluckyCroco](https://osu.ppy.sh/users/9461160) & 
[Amiichii](https://osu.ppy.sh/users/9228569)

We’d like to use this section as a reminder that you can apply as a Beatmap Nominator at any time over at [BN/NAT website](https://bn.mappersguild.com/) as long as you reach the activity requirements. We hope to see some of your names on this list next month!

### BN Retirements

Next, we have those who have taken their leave from the group. We’d like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!

**In osu!:** 
[Trynna](https://osu.ppy.sh/users/2652951)

**In osu!taiko:** 
[qoot8123](https://osu.ppy.sh/users/766371)

**In osu!catch:** 
[Sanyi](https://osu.ppy.sh/users/7496029), 
[Du5t](https://osu.ppy.sh/users/6053071) & 
[Rocma](https://osu.ppy.sh/users/566276)

**In osu!mania:** 
[Tailsdk](https://osu.ppy.sh/users/6751666), 

> [Critical_Star](https://osu.ppy.sh/users/3793196) & 

Reviewer: Add [MEGAtive](https://osu.ppy.sh/users/3094101) since they just resigned

- [**Background Quality Guideline**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.5): A new guideline has been added specifying that background images must be of reasonable quality.

### osu!taiko

- **Drum Sampleset**: A rule was removed which forbade the use of the drum sampleset. Previously, the game had no drum sample set support for osu!taiko, but this was fixed and samples added. Thus, the rule has been removed as the sampleset is now usable.

### osu!catch

- [**HP Difficulty Guidelines**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!catch#difficulty-specific): The difficulty specific guidelines for HP settings have been adjusted for Cup to Overdose to be in line with community standards.

### osu!mania

- [**Hitsound Additions Requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.4): Following this change, osu!mania beatmapsets containing only difficulties of Insane or above do not require hitsound additions. 

- [**Difficulty name requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!mania#rules):
A new specification has been added that sets with a single key mode must not use the key mode in the difficulty name.

> 

Reviewer 2: celerih says points should probably be spaced here as in the general section (i.e. no newlines between each)

- [**Long note changes in Easy and Normal**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!mania#-easy):
The previous guidelines in Easy and Normal that "A long note cannot be released during another long note's body" have been removed. They have been replaced with "long note releases should be at least 1/1 beat from each other" for Easy difficulties and "long note releases should be at least 1/2 beat from each other" for Normal difficulties.

## <a id="natmeetings"></a>NAT Meetings

> The past 2 months have been filled with public meetings organized by the NAT, aimed at improving the ranking system based on community input. A wide range of topics such as BN/NAT activity requirements and improving the transparency of the application process to the two groups has been covered so far, and there is more to come! If you wish to participate in future meetings, join the [osu!dev discord server](https://discord.gg/ppy) and go through the verification process.

Reviewer 3: ```suggestion
The past 2 months have been filled with public meetings organized by the NAT, aimed at improving the ranking system based on community input. A wide range of topics such as BN/NAT activity requirements or transparency of the application process for both groups have been covered so far, and there is more to come! If you wish to participate in future meetings, join the [osu!dev discord server](https://discord.gg/ppy) and go through the verification process.
```
would also mention that meetings are taking place on ``#modding`` channel

Upcoming meetings will be announced with a specific time and agenda in the #modding channel of the osu!dev Discord server, as well as in an announcement post in the [Mapping Discussion](https://osu.ppy.sh/community/forums/56) forum. The next meeting is scheduled for 1st of March, but the topic and agenda are still being worked on and will be announced soon.	

### [Jan 3rd, 2020 - Promotion of NAT Members](https://osu.ppy.sh/community/forums/topics/1006119)

The most important information which was discussed during this meeting was the addition of new NAT and its team size. The NAT wiki article has also been updated to include relevant information such as how the team size of the NAT depends on what is needed and in what work fields. If more help in one work field is required, a new NAT will be considered. 

### [Jan 18th, 2020 - Application Evaluation](https://osu.ppy.sh/community/forums/topics/1012519)

> The BN application mod requirements have been made more clear, and the BN website also received several improvements such. One addition was a “Would you nominate this map, if no why?” checkbox. The modder can explain in this section whether they would nominate the beatmaps which they provided as modding examples. All this information helps the NAT and evaluating BNs to make a fair evaluation of the applicant. 

Reviewer 3: ```suggestion
The BN application mod requirements have been made more clear, and the BN website also received several improvements, the biggest one being a “Would you nominate this map. If not, why?” checkbox on the application form. It vastly improves the fair evaluation system.
```
Shortened this one as the purpose of checkbox thing is pretty much self-explanatory so there's probably no point in giving it such a complex description.

**In osu!:** 
[celerih](https://osu.ppy.sh/users/4696296), 
[SMOKELIND](https://osu.ppy.sh/users/9327302) & 
[Stack](https://osu.ppy.sh/users/6122935)

**In osu!taiko:** 
[\_DUSK\_](https://osu.ppy.sh/users/6092181), 
[Capu](https://osu.ppy.sh/users/2474015), 
[Gamelan4](https://osu.ppy.sh/users/9856910), 
[Hoshikuz](https://osu.ppy.sh/users/9892196), 
[incandescence](https://osu.ppy.sh/users/6256027), 
[kanpakyin](https://osu.ppy.sh/users/394326), 
[Stefan](https://osu.ppy.sh/users/626907), 
[Vulkin](https://osu.ppy.sh/users/4901066) & 
[Yales](https://osu.ppy.sh/users/2377881)

**In osu!catch:** 
[Lacrima](https://osu.ppy.sh/users/4915649) & 
[Dapuluous](https://osu.ppy.sh/users/8140944)

**In osu!mania:** 
[Voxa](https://osu.ppy.sh/users/9229539), 
[Dubstek](https://osu.ppy.sh/users/9555243), 
[UnluckyCroco](https://osu.ppy.sh/users/9461160) & 
[Amiichii](https://osu.ppy.sh/users/9228569)

> We’d like to use this section as a reminder that you can apply at any time over at https://bn.mappersguild.com/ as long as you reach the activity requirements. We hope to see some of your names in this list next month!

Reviewer 3: ```suggestion
We’d like to use this section as a reminder that you can apply as a Beatmap Nominator at any time over at [BN/NAT website](https://bn.mappersguild.com/) as long as you reach the activity requirements. We hope to see some of your names on this list next month!
```

Reviewer 2: would add `the` before `BN/NAT website` as well

### BN Retirements

> Next we have those who have taken their leave from the group. We’d like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!

Reviewer 3: ```suggestion
Next, we have those who have taken their leave from the group. We’d like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!
```

**In osu!:** 
[Trynna](https://osu.ppy.sh/users/2652951)

**osu!taiko:** 
[qoot8123](https://osu.ppy.sh/users/766371)

**In osu!catch:** 
[Sanyi](https://osu.ppy.sh/users/7496029), 
[Du5t](https://osu.ppy.sh/users/6053071) & 
[Rocma](https://osu.ppy.sh/users/566276)

**In osu!mania:** 
[Tailsdk](https://osu.ppy.sh/users/6751666), 
[Critical_Star](https://osu.ppy.sh/users/3793196) & 
[Kyousukee](https://osu.ppy.sh/users/8842107)

### NAT Additions

> In addition to the new BNs recently added, some new NAT members have been brought on! 

Reviewer 3: ```suggestion
In addition to the recently added BNs, some new NAT members have been brought on! 
```

> The first addition we got on the 6h January with **[yaspo](https://osu.ppy.sh/users/4945926)**! Being previously roughly over a year a Beatmap Nominator, he will now support the NAT for the osu! game mode. He will mainly help with the BN evaluations and applications. But he has also shown already himself with leading through the NAT meetings! So welcome aboard yaspo!

Reviewer 3: ```suggestion
The first addition we got on the 6h January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!
```

Reviewer 2: `6h` -> `6th` also

Our second addition we got on the 26th January is **[Greaper](https://osu.ppy.sh/users/2369776)**! With being previously a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications and make sure, the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!

## <a id="mapping_contests"></a>Mapping Contests

> If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it to [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month’s installment of the Mappers’ Report. We are looking to highlight more mapping contests from the community!

Reviewer 3: ```suggestion
If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month’s installment of the Mappers’ Report. We are looking to highlight more mapping contests from the community!
```

For the avid fruit catchers of the world, the 6th osu!Mapping Olympiad has shown itself, a great opportunity to flex your fruit placing muscles! This osu!catch contest requires you to map 2 difficulties for MYLK’s Waffle Wave track. Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-15-osu-mapping-olympiad-6-catch) for all the details! As this is an olympiad the winner of the tournament will be given the title of Elite Mapper. Good luck to all of those participating, you have until March 13th to submit your fruity feasts.

### Newspaper Cup

The fantastic Chinese community organized tournament **Newspaper Cup** has made its yearly appearance and has now begun! This will be the contest’s 6th yearly installment, making it one of the longest standing community organized mapping contests. All of the relevant information can be found [in this forum post](https://osu.ppy.sh/community/forums/topics/1023671) in both Chinese and English. You have until **March 15th** to submit your entries for one or more of the 4 available songs. Good luck to those planning on entering their creations into the contest!

## <a id=”summary”></a>Summary

> Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](discord.gg/ppy) and to do your best on all those fantastic mapping contests open to you! Thank you for your time reading this quite hefty article. As this is our first installment of this report any feedback is appreciated as we are still working hard at setting this up. Additionally the next installments should be a bit smaller, as we had a lot to cover this time. We hope the information will prove itself useful to you!

Reviewer 3: ```suggestion
Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](discord.gg/ppy) and to do your best on all those fantastic mapping contests open for your submissions! Thank you for your time reading this quite hefty article. As this is our first installment of the Mappers' Report, any feedback is appreciated as we are still working hard at setting this up. Additionally, the next installments should be a bit smaller, as we had a lot to cover this time. We hope the information will prove itself useful to you!
```
